OrbitClientServices/include/OrbitClientServices/ProcessManager.h

```diff
@@ -35,17 +35,19 @@
 //
 class ProcessManager {
  public:
+  using ProcessList = std::vector<orbit_grpc_protos::ProcessInfo>;
```

Comment:
maybe keep vector in definitions? this using is hiding information about the type... I almost confused it with class we used to have in OrbitCore.

Reply:
I think it's more readable, and there's IDEs to display type info ;-) But in all seriousness, I don't have a strong opinion, I wasn't touching this part of the code so much before, so if it's confusing I'll keep the std::vector<> in the definitions.